redbot/cogs/mutes/mutes.py

```diff
@@ -1726,7 +1729,7 @@ async def channel_mute_user(
 
         if channel.id not in self._channel_mutes:
             self._channel_mutes[channel.id] = {}
-        current_mute = self._channel_mutes.get(channel.id)
+        current_mute = self._channel_mutes.get(channel.id) or None
```

Comment:
Hmm, this seems like another bug - this should be:
```suggestion
        current_mute = self._channel_mutes[channel.id].get(user.id)
```